### PR TITLE
rework nextKatacodaStep

### PR DESCRIPTION
**Pull request by EduardKrieger**

### CODE REVIEW

**Review by denise-khuu — APPROVED** [review]

File: runners/wikiConsole/index.ts [hunk 1]
```diff
@@ -155,25 +155,33 @@ export class WikiConsole extends WikiRunner {
               return null;
     }
 
-    runNextKatacodaStep(runCommand: RunCommand): RunResult {
+    runDisplayContent(runCommand: RunCommand): RunResult {
         let tempFile = path.join(this.getTempDirectory(), runCommand.command.name + ".md");
         fs.writeFileSync(tempFile, "");
+        let counter = 2;
```
Comment (GuentherJulian) [inline runners/wikiConsole/index.ts:161]: You set `counter` to 2 two times. Move it to the beginning of the else-if-block, then you only need it once

File: runners/wikiConsole/index.ts [hunk 2]
```diff
@@ -155,25 +155,33 @@ export class WikiConsole extends WikiRunner {
...
             } else if (param.image) {
-                let image = path.join(this.playbookPath, param.image);
-                fs.appendFileSync(tempFile, "![" + path.basename(image) + "](./assets/" + path.basename(image) + ")");
+                this.createFolder(path.join(this.outputPathTutorial, "images"), false);
+                let imageName = path.join(this.outputPathTutorial, "images",path.basename(param.image));
+                while(fs.existsSync(imageName)){
+                    imageName = path.join(this.outputPathTutorial, "images",counter+"_"+path.basename(param.image));
+                    counter++;
+                }
+                fs.createFileSync(imageName);
```
Comment (GuentherJulian) [inline runners/wikiConsole/index.ts:175]: Is this line really needed?

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/wikiConsole/index.ts (hunk 1 above)
Comment (EduardKrieger) [reply runners/wikiConsole/index.ts:187]: Removed the second assignment

File: runners/wikiConsole/index.ts (hunk 2 above)
Comment (EduardKrieger) [reply runners/wikiConsole/index.ts:201]: Removed it.

**Review by GuentherJulian — APPROVED** [review]

**Review by denise-khuu — APPROVED** [review]

**Pull request merged**